### PR TITLE
Wait for configuration update in testEventLoggingLogModeUpdate2

### LINKED ISSUE #14784: Test Failure: com.ibm.ws.event.logging.FATTest.testEventLoggingLogModeUpdate2
RTC 279059

```
testEventLoggingLogModeUpdate2:junit.framework.AssertionFailedError: 2020-10-10-04:02:45:017 BEGIN records FOUND for logMod Exit!
at com.ibm.ws.event.logging.FATTest.testEventLoggingLogModeUpdate2(FATTest.java:699)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:197)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:318)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:171)
```

### PR DESCRIPTION
**Pull request by Yushan-Lin**
Fixes #14784 
Test fails when ran in different order. BEGIN logs are read from log before server configuration is updated.

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
#build

File: dev/com.ibm.ws.event.logging_fat/fat/src/com/ibm/ws/event/logging/FATTest.java [hunk 1]
```diff
@@ -680,6 +680,7 @@ public void testEventLoggingLogModeUpdate3() throws Exception {
     @Test
     public void testEventLoggingLogModeUpdate2() throws Exception {
         server.setServerConfigurationFile("server_logModeExit.xml");
+        server.waitForStringInLog("CWWKG0017I", 90000);
```
Comment (fmhwong) [inline dev/com.ibm.ws.event.logging_fat/fat/src/com/ibm/ws/event/logging/FATTest.java:683]: Is "CWWKG0017I" mandatory?  If yes, please surround with `assertNotNull()`

Comment (Yushan-Lin) [reply dev/com.ibm.ws.event.logging_fat/fat/src/com/ibm/ws/event/logging/FATTest.java:684]: Updated PR

**Review by fmhwong — APPROVED** [review]
LGTM

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_xVKtkCULEeux8uC7tiSOag

**Comment by Yushan-Lin** [comment]
#build

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Pull request merged**